tests/perf/main/install-many-snaps-no-wait/task.yaml

```diff
@@ -0,0 +1,62 @@
+summary: Ensure the system handles properly a installing many snaps with --no-wait parameter
+
+details: |
+    Install different snaps many times based on arch availability. In this
+    scenario all the snaps are installed in parallel and the test checks
+    the system can handel all the installs and ramains running.
+
+kill-timeout: 30m
+
+execute: |
+    if [ -z "$NUM_SNAPS" ]; then
+        NUM_SNAPS=20
+    fi
+
+    LETTERS="a b c d e f g h i j k l m n o p q r s t u v w x y z"
+    INCLUDED=0
+    CHANNEL="stable"
+    SNAP_LIST=""
+
+    # shellcheck disable=SC2086
+    for letter in $LETTERS; do
+        if [ "$INCLUDED" = "$NUM_SNAPS" ]; then
+            echo "already $NUM_SNAPS ready to install, now check other features"
+            break
+        fi
+
+        snaps="$(snap find --narrow "$letter")"
+        SNAP_NAMES="$(echo "$snaps" | awk '{if($4~/-/){print $1}}' | tail -n+2)"
+        for SNAP in $SNAP_NAMES; do
+            # Get the info from latest/$CHANNEL
+            # shellcheck disable=SC2153
+            if ! CHANNEL_INFO="$(snap info --unicode=never "$SNAP" | grep " latest/$CHANNEL: ")"; then
+                echo "Snap $SNAP not found"
+                continue
+            fi
+            if echo "$CHANNEL_INFO" | MATCH "$CHANNEL:.*-$"; then
+                SNAP_LIST="$SNAP_LIST $SNAP"
+                INCLUDED=$(( INCLUDED + 1 ))
+            fi
+
+            if [ "$INCLUDED" = "$NUM_SNAPS" ]; then
+                echo "already $NUM_SNAPS included in the list"
+                break
+            fi
+        done
+    done
+
+    echo "Installing snaps: $SNAP_LIST"
+    # shellcheck disable=SC2086
+    for SNAP in $SNAP_LIST; do
+        snap install --no-wait "--$CHANNEL" "$SNAP"
+    done
+
+    while true; do
+        if snap changes | NOMATCH " Doing.*Install \".*\" snap"; then
+            break
+        fi
+        sleep 1
+    done
```

Comment:
We should probably have a retry limit here, even if it's large since there so many snaps. Also can we use `retry` to turn this into a one liner?

Reply:
well, I avoided the retry because in real devices (slow) it could take a long time installing and in the end of the while will be the kill-timeout of the test.

Reply:
Ah I see, thanks for the explanation